components/sysutils/slurm/files/getgrouplist.h

```diff
@@ -0,0 +1,96 @@
+/*-
```

Comment:
Right, I always forget that the relationship is unidirectional...
The reason is that illumos does not have getgrouplist() maybe I should file a RFE.